content/post/kpis.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "KPI Principles"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["analytics", "KPI"]
+date: 2019-03-30T21:19:01-04:00
+draft: true
+---
+
+Everyone wants to know how their business is doing. They also want to know what the levers available to them are, and how to see if those levers make a difference. Enter the Key Performance Indicator (KPI). KPIs are a common tool for quantifying the performance of your business. They're also a symbol for folks in the organization to rally around and anchor against, providing clarity and aligning cross departmental objectives.
```

Comment:
I think this lead-in paragraph could be tightened up a bit.

When I think of KPIs I think of them as a _management tool_ for monitoring and _improving_ the business. I think if you anchor more of this post around management and continuous improvement it might flow a little better.

(That's just my idea!)